remoteappmanager/application.py

```diff
@@ -1,11 +1,15 @@
 import os
-from tornado import web, gen
 from urllib import parse
-import tornado.ioloop
 
+from remoteappmanager.db.orm import transaction
```

Comment:
A minor (subjective?) comment, could do a bit of ordering for the imports?
[standard libraries]
[other libraries]
[local package]